static/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,22 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-to-show">
+                    <div class="view_all_stream_button_container">
```

Comment:
Not sure we should use button in this class name since there are no buttons to show.

Reply:
I added `view_all_stream_link_container` class.
Is that okay or we should use something else.